server/consumer.go

```diff
@@ -272,6 +272,10 @@ var (
 	AckNext = []byte("+NXT")
 	// Terminate delivery of the message.
 	AckTerm = []byte("+TERM")
+
+	// reasons to supply when terminating messages using limits
+	ackTermLimitsReason        = "Message deleted by stream limits"
+	ackTermUnackedLimitsReason = "Unacknowledged message deleted by stream limits"
```

Comment:
Could be user initiated, think purge. So would just do ""Unacknowledged message was deleted"

Reply:
OK pushed an update